oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
 # This is a temporary fix for a Google internal issue.
 try:
-    from google.appengine.ext import ndb
```

Comment:
Ahhh got it. I'd imagine the comment above is for "someone else's library" but here we are in full control. You can find examples of the BDFL doing this in real life ([e.g.][1] in `ndb`).

[1]: https://github.com/GoogleCloudPlatform/datastore-ndb-python/blob/ecc346e86b2b3790ef82c2454d03a789de24030f/ndb/__init__.py#L22